purchases/src/main/kotlin/com/revenuecat/purchases/strings/BillingStrings.kt

```diff
@@ -22,4 +22,9 @@ internal object BillingStrings {
     const val BILLING_CLIENT_RETRY = "Retrying BillingClient connection after backoff of %s milliseconds."
     const val ILLEGAL_STATE_EXCEPTION_WHEN_CONNECTING = "There was an IllegalStateException when connecting to " +
         "BillingClient. This has been reported to occur on Samsung devices on unknown circumstances.\nException: %s"
+    const val BILLING_CONNECTION_ERROR_INAPP_MESSAGES = "Error connecting to billing client to display " +
+        "inapp messages: %s"
+    const val BILLING_INAPP_MESSAGE_NONE = "No Google Play in-app message was available."
+    const val BILLING_INAPP_MESSAGE_UPDATE = "Subscription status was updated from In-App Message."
```

Comment:
we are writing in-app in three different ways in this strings file `inapp`, `in-app`, `In-App`